annot8-components-files/src/main/java/io/annot8/components/files/sinks/PhraseToCSV.java

```diff
@@ -0,0 +1,209 @@
+/* Annot8 (annot8.io) - Licensed under Apache-2.0. */
+package io.annot8.components.files.sinks;
+
+import de.siegmar.fastcsv.writer.CsvWriter;
+import io.annot8.api.annotations.Annotation;
+import io.annot8.api.annotations.Group;
+import io.annot8.api.capabilities.Capabilities;
+import io.annot8.api.components.annotations.ComponentDescription;
+import io.annot8.api.components.annotations.ComponentName;
+import io.annot8.api.components.annotations.SettingsClass;
+import io.annot8.api.components.responses.ProcessorResponse;
+import io.annot8.api.context.Context;
+import io.annot8.api.data.Content;
+import io.annot8.api.data.Item;
+import io.annot8.api.settings.Description;
+import io.annot8.common.components.AbstractProcessor;
+import io.annot8.common.components.AbstractProcessorDescriptor;
+import io.annot8.common.components.capabilities.SimpleCapabilities;
+import io.annot8.common.data.bounds.SpanBounds;
+import io.annot8.common.data.utils.SortUtils;
+import io.annot8.conventions.GroupTypes;
+import io.annot8.conventions.PropertyKeys;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@ComponentName("Phrases to CSV")
+@ComponentDescription(
+    "Appends phrase groups to a CSV in the format document,phraseType,value,start,end")
+@SettingsClass(PhraseToCSV.Settings.class)
+public class PhraseToCSV
+    extends AbstractProcessorDescriptor<PhraseToCSV.Processor, PhraseToCSV.Settings> {
```

Comment:
Could be more versatile to go to a Table content. then write to file if needed.

Reply:
No need to do this though, just a comment.

Reply:
A good suggestion - done